crates/pixi_manifest/src/toml/build_backend.rs

```diff
@@ -179,15 +158,6 @@ mod test {
         insta::assert_debug_snapshot!(parsed);
     }
 
-    #[test]
-    fn test_disallow_source() {
-        assert_snapshot!(expect_parse_failure(
-            r#"
-            backend = { name = "foobar", git = "https://github.com/org/repo" }
-        "#
-        ));
-    }
-
```

Comment:
Maybe then we should expect it to parse, rather than removing test?

Reply:
These snapshot tests are there to snapshot the error message that is reported. 

I want to add a full end-2-end test that should cover this. Or do you think we should also add explicit parsing (unit) tests?

Reply:
Actually, don't think that it worth retesting this if you'll add end-2-end test in this PR.